.github/workflows/create-tag.yml

```diff
@@ -131,3 +135,13 @@ jobs:
 
           gh repo set-default ${{ github.repository }}
           gh release create "$TAG" --verify-tag --draft --title "$TAG" -F "$TEMPFILE"
+
+      - name: Create PR updating dependencies
+        if: ${{ github.event.inputs.update_deps == 'true' }}
+        run: |
+          make update-dependencies
```

Comment:
probably also run `make go-generate` here?
I need to regenerate some mocks (common/archiver/s3store/mocks/generate.go) in https://github.com/temporalio/temporal/pull/5789

Reply:
the other option would be to have update-dependencies depend on go-generate in the makefile. just mentioning, I think I like this better